src/py/flwr/common/constant.py

```diff
@@ -120,3 +120,27 @@ class ErrorCode:
     def __new__(cls) -> ErrorCode:
         """Prevent instantiation."""
         raise TypeError(f"{cls.__name__} cannot be instantiated.")
+
+
+class Status:
+    """Run status."""
+
+    STARTING = "starting"
```

Comment:
I find the `STARTING` status quite confusing. Does it mean that there is a duration between starting a run until running a run?
Also, is it a good idea that we use a status `RUNNING` for a run?

Reply:
Yes, it's actually described in the [RFC](https://www.notion.so/flowerlabs/Introduce-Run-status-3a131d67488c4226877d986a0e6a8550). It's the time when the superlink receives the `CreateRun` request from the SuperLink but the ServerApp process hasn't started yet.